apm-agent-core/src/main/java/co/elastic/apm/agent/impl/ElasticApmTracer.java

```diff
@@ -383,20 +384,27 @@ public void endSpan(Span span) {
             span.decrementReferences();
             return;
         }
-        if (!span.isComposite()) {
+        SpanConfiguration spanConfiguration = getConfig(SpanConfiguration.class);
```

Comment:
why does this have wider scope than the following if()?

Reply:
No specific reason. I will change it.